indico_jacow/templates/custom_affiliation.html

```diff
@@ -0,0 +1,16 @@
+{% if person.jacow_affiliations -%}
```

Comment:
Hmm maybe move that into the python code since you don't return anything if there aren't any jacow affiliations?